### ISSUE TITLE
Magit stickers are now available for everyone

### ISSUE THREAD

**Issue by tarsius** [magit/magit#3547]
Some users who missed the Kickstarter campaign have asked whether they can still buy some stickers. Now you finally can! I don’t have that many, so better hurry. The price is the same as if you backed the crowdfunding campaign, which isn't cheap, but remember you are primarily paying for development time.

If you have donated elsewhere in the past, then you can also get some stickers for free. Again, act fast.

Please see https://magit.vc/buy/ for more information.

Cheers, Jonas

Ps. Mostly opening this here to ask: @mvollmer @kyleam @npostavs @dudebout @vanicat @sigma @philjackson @pjweisberg would you like some free stickers?

**Comment by mvollmer** [comment]
> would you like some free stickers?

Yeah, stickers! :-)  I missed the kickstarter, and I actually don't mind paying.  I'll buy some from your web page. Would be cool if you could put your autograph on them maybe.

**Comment by sigma** [comment]
much appreciated, but I'm good. I might order some later if I change my mind :)

**Comment by tarsius** [comment]
@mvollmer Only if you sign some yourself and send them back to me :stuck_out_tongue_winking_eye: 

Going to the post office today.

**Comment by dudebout** [comment]
Thanks for the offer. Got mine from the Kickstarter campaign.
